docs/IsolationNetworkPolicies.md

```diff
@@ -0,0 +1,105 @@
+# Isolation 
+
+_**Note:**_ below we start with Network Polices as an isolation method, later we will add another methods.
+
+## Introduction
+
+In the current Fybrik implementation the entry point of a data plane is published as a part of the FybrikApplication 
+status, it is completely non-protected, and as result everyone can connect to the data plane. 
+In “Read” flow cases it leads to governance/security violations, “Write” and “Delete” flows can create severe accidents, 
+such as unauthorized data changes or deletions. Given that we need to prevent unauthorized data plane access. 
+
+_Note:_ the same situation exists between chained modules.  
+
+## Challenge
+
+When we talk about accessing Fybrik data plane, we are talking about connectivity between a user workload, which is out of 
+Fybrik scope, and Fybrik module, which can be developed by a third party. Therefore, in this document (and in its 
```

Comment:
I agree that the highest priority is to provide isolation between the workload and the module(s) with which it interfaces.  However, I believe we also need to provide isolation between modules - even those that do not interact directly with the workload.  Right?

Reply:
Absolutely, and it is explained in the document

Reply:
Shouldn't it be included here in the challenge part?  Here you only mention isolating the connection between the workload and the module(s) it interacts with

Reply:
OK, added it